Mathlib/Combinatorics/SimpleGraph/UniversalVerts.lean

```diff
@@ -51,4 +52,23 @@ lemma Subgraph.IsMatching.exists_of_universalVerts [Finite V] {s : Set V}
   have hadj (v : s) : G.Adj v (f v) := ht.1 (f v).2 (hd.ne_of_mem (f v).2 v.2)
   exact Subgraph.IsMatching.exists_of_disjoint_sets_of_equiv hd.symm f hadj
 
+lemma disjoint_image_val_universalVerts (s : Set G.deleteUniversalVerts.verts) :
+    Disjoint (Subtype.val '' s) G.universalVerts := by
+  simpa [deleteUniversalVerts, Subgraph.deleteVerts_verts, ← Set.disjoint_compl_right_iff_subset,
+    Set.compl_eq_univ_diff] using Subtype.coe_image_subset _ s
+
+/-- In this lemma we consider components after deleting universal vertices. If we take
+one such component and remove both a set of representatives of odd components and a subset
+of universal vertices, then an even number of vertices remain. -/
```

Comment:
Similarly I think you could rephrase this docstring to be more understandable on hover

Reply:
Improved:)